lib/internal/cluster/master.js

```diff
@@ -12,6 +12,18 @@ const cluster = new EventEmitter();
 const intercom = new EventEmitter();
 const SCHED_NONE = 1;
 const SCHED_RR = 2;
+var debugPortOffset = 0;
+
+const debugArgRegex = /^(--inspect|--debug|--debug-(?:brk|port))/;
+
+const hasDebugArg = process.execArgv.some((argv) => {
+  return debugArgRegex.test(argv);
+});
```

Comment:
Nit: If you want, this can be more concise:  

```js
const hasDebugArg = process.execArgv.some((argv) => debugArgRegex.test(argv)); 
```
Both the more verbose form currently in the PR and this shorter form appear in the current code base, but the above shorter form is the more common way it shows up. As such, I have a *slight* preference for it as it will probably be the way things end up if we ever enforce arrow function style with a lint rule (which I hope we will at some point).